portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro

```diff
@@ -14,6 +14,64 @@ definition {
 		AssertConfirm(value1 = "Are you sure you want to deactivate this?");
 	}
 
+	macro filtersApp {
+		var key_filterName = "${filterName}";
+
+		SelectFrame.selectFrame(locator1 = "IFrame#MARKETPLACE_STORE_PORTLET_IFRAME");
+
+		WaitForElementPresent(
+			key_filterName = "${filterName}",
+			locator1 = "AppManager#APP_FILTER_LABEL");
+
+		if (isSet(filterBy)) {
```

Comment:
Not sure if this can be replace by the exist macro `setFilterAndOrder`, if not, no need to use if here, since this is a macro for filter.

Reply:
Can not I have tried